.Documentation/articles/innoactive-creator/default-conditions.md

```diff
@@ -23,149 +25,281 @@ See [this article](step-inspector.md) to learn Step Inspector controls.
+
+Before the controller button is pressed, by default, the object is already highlighted to indicate that the trainee can now grab this object. The training designer can change this in the interactable highlighter script of the given object. You can enable/disable this interaction effect, change the highlight color and transparency for `On Touch`, `On Grab`, and `On Use`.
+
+
+#### Interaction in Desktop Mode
+
+To grab the object in Desktop Mode, the trainee has to move the mouse over the object (so that the ray hits the object's collider) and press the left mouse button.
+
+Before the mouse button is pressed, by default, the object is already highlighted to indicate that the trainee can now grab this object. This “on hover” effect can be changed by the template developer in the `Hover Highlight Handler` of the Desktop Mode Rig prefab. 
 
-### Configuration
 
-- #### Grabbable object
 
-    This field contains the `Training Scene Object` with a `Grabbable Property` that is required to be grabbed.
-    Besides, the object must have a collider component in order to be grabbed.
+### Application Example
 
-    [![Grabbable Properties](../images/conditions/grabbable_properties.png "")](../images/conditions/grabbable_properties.png)
+- Trainees need to grab objects to position them somewhere.
+- Trainees need to grab a tool to use it.
+- Trainees need to grab a component part to assemble it to another component part.
 
+
+### Configuration
+
+- #### Object
+
+    The `Training Scene Object` to grab. The object needs to have the `Grabbable Property` and a collider component configured. The collider defines the area where the trainee can grab this object.
+
+### Location of this Condition (for Developers)
+This behavior is part of the <a href="https://github.com/Innoactive/Basic-Interaction-Component" target="_blank">Basic-Interaction</a> component. The file is located <a href="https://github.com/Innoactive/Basic-Interaction-Component/blob/develop/Runtime/Conditions/GrabbedCondition.cs" target="_blank">here</a>.
+ 
 ------
 
 ## Move Object into Collider
 
 ### Description
 
-The `Move Object into Collider` condition is fulfilled if the target object is within the specified collider for the required amount of time while this condition is active.
+This condition is fulfilled when the `Object` is within the specified `Collider` for the required amount of time (`Required seconds inside`) while this condition is active.
+
+#### Interaction in VR
+In order to move the object into a collider, trainees need to move their hands/controllers into the object (its collider) and press the trigger button. While holding the trigger button and moving the arm, the object stays grabbed by trainees hands. Trainees move the object into the specified collider and release it. The button to trigger can be changed by the template developer as described <a href="https://developers.innoactive.de/documentation/creator/latest/articles/xr/index.html" target="_blank">here</a>.
```

Comment:
```suggestion
In order to move the object into a collider, the trainee will do the following steps. 
1. Grab the object
2. Keep holding the button pressed to keep holding the object
3. Move towards the collider
4. Move the object into the specified collider and release it by releasing the grab button
The button to trigger can be changed by the template developer as described <a href="https://developers.innoactive.de/documentation/creator/latest/articles/xr/index.html" target="_blank">here</a>.
```
We just explained grabbing before, let's not do that all over again :) Additionally, we probably should explain how to keep holding an object (by not releasing the grab button) in the grab condition instead of explaining it here. I think it makes more sense there.
And I suggest this with numbers here to make it easier to read.

Reply:
since users do not read this documentation from top to down, it is ok to be repetetitive.